### DISCUSSION TITLE
Features for journal

### DISCUSSION THREAD

**Discussion by Slabhub** [YU000jp/logseq-plugin-default-template#3] [Ideas]
Plugin works for journal page  if you click journal page not from Journal section. 
Maybe  it can be added  special journal  Default template  or  revised advanced sections.

**Comment by YU000jp** [comment]
This plugin is designed to apply a template when a single page is opened and there is only one block and it is empty. "Journals" is not assumed. In that case, use [More journal template plugin](https://github.com/YU000jp/logseq-plugin-weekdays-and-weekends).
However, thanks to you, I realized that I had not assumed that the template would be applied to daily journal. If the user has not set up a journal template, ~~it will be applied when the journal single page is opened~~.
